cluster/gce/gci/configure-helper.sh

```diff
@@ -2104,14 +2104,17 @@ EOF
     prepare-kube-proxy-manifest-variables "$src_dir/kube-proxy/kube-proxy-ds.yaml"
     setup-addon-manifests "addons" "kube-proxy"
   fi
+  if [[ "${ENABLE_CLUSTER_MONITORING:-}" != "none" ]]; then
```

Comment:
I prefer `if [[ "${ENABLE_CLUSTER_MONITORING:-}" == "prometheus" ]]`. It makes sense to have fully separated conditions for prometheus and other monitoring systems, because this is the only one that doesn't use heapster. Please add some comments to make this separation clear, e.g. "set up cluster monitoring using prometheus" and "set up cluster monitoring using heapster"